platform/commonUI/inspect/src/gestures/InfoGesture.js

```diff
@@ -131,6 +126,13 @@ define(
             this.pendingBubble = this.$timeout(displayBubble, this.delay);
 
             this.element.on('mouseleave', this.hideBubbleCallback);
+
+            // Also make sure we dismiss bubble if representation is destroyed
+            // before the mouse actually leaves it
+            this.scopeOff =
+                this.element.scope() &&
```

Comment:
This is to handle the case where mct-representation is removed from the DOM-- right? In this case, shouldn't the gesture service skip attaching gestures to invalid elements entirely?